packages/react-examples/src/react-charting/LineChart/LineChart.Multiple.Example.tsx

```diff
@@ -63,16 +64,45 @@ export class LineChartMultipleExample extends React.Component<{}, ILineChartMult
     }
   };
 
+  private _formatDate = (date: Date, useUTC?: boolean) => {
+    const timeFormat = useUTC ? d3UtcFormat : d3TimeFormat;
+    return timeFormat('%-e %B %Y, %H:%M:%S')(date) + (useUTC ? ' GMT' : '');
+  };
+
   private _styledExample(): JSX.Element {
     const points: ILineChartPoints[] = [
       {
         data: [
-          { x: new Date('2018/01/01'), y: 10, xAxisCalloutData: '2018/01/01', yAxisCalloutData: '10%' },
-          { x: new Date('2018/02/01'), y: 30, xAxisCalloutData: '2018/02/01', yAxisCalloutData: '18%' },
-          { x: new Date('2018/03/01'), y: 10, xAxisCalloutData: '2018/03/01', yAxisCalloutData: '24%' },
-          { x: new Date('2018/04/01'), y: 30, xAxisCalloutData: '2018/04/01', yAxisCalloutData: '25%' },
-          { x: new Date('2018/05/01'), y: 10, xAxisCalloutData: '2018/05/01', yAxisCalloutData: '15%' },
-          { x: new Date('2018/06/01'), y: 30, xAxisCalloutData: '2018/06/01', yAxisCalloutData: '30%' },
```

Comment:
why remove y axis callout data field.

Reply:
This field is not present in all other legends. 
Either we can assign this field to 10(not 10%) so that the format remains same or leave it empty.

Reply:
this field is intentional. dont remove it.

Reply:
Oh okay.